datumaro/components/extractor.py

```diff
@@ -170,6 +241,9 @@ def get(self, id, subset=None):
                 return item
         return None
 
+    def media_type(self):
```

Comment:
Probably, it would be reasonable to add to `IExtractor` also.

Reply:
Added this to `IExtractor`